.docker/nginx/nginx.conf

```diff
@@ -0,0 +1,26 @@
+user  nginx;
+worker_processes  4;
+daemon off;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    access_log  /var/log/nginx/access.log;
+    # Switch logging to console out to view via Docker
+    #access_log /dev/stdout;
```

Comment:
We should think Docker-way only these days.

Reply:
I don't understand, please explain.

Reply:
Uncomment `access_log /dev/stdout;` a drop logging to file.